lib/IRGen/GenDecl.cpp

```diff
@@ -1822,8 +1822,8 @@ void irgen::updateLinkageForDefinition(IRGenModule &IGM,
   // entire linkage computation.
   UniversalLinkageInfo linkInfo(IGM);
   auto linkage =
-      getIRLinkage(linkInfo, entity.getLinkage(ForDefinition),
-                   ForDefinition, entity.isWeakImported(IGM.getSwiftModule()));
+      getIRLinkage(linkInfo, entity.getLinkage(ForDefinition), ForDefinition,
+                   entity.isWeakImported(IGM.getSwiftModule()));
```

Comment:
This is no longer a real change; please revert it to make the patch simpler.

Reply:
OK